cl/search/templates/search.html

```diff
@@ -274,7 +274,7 @@ <h2 id="result-count" class="bottom">
             {% include "includes/no_results.html" %}
         {% endif %}
 
-        {% include "includes/pagination.html" %}
+        {% include "includes/new_pagination.html" with page_obj=results only%}
```

Comment:
We cannot allow users to go to the last page of search results (it's a performance nightmare). Let's be sure to disable that.

Reply:
You have said this before, but I don't understand how the last page is any more expensive than any other page.  Regardless of which page you go to you are going to have to apply the query filter, WHERE in the SELECT statement, after that you are going to have to sort the results.  Finally, you do the magic where you see which rows you want returned.

`ASC LIMIT 5 OFFSET 15` is what the ORM generates for `queryset[15:20]`

Given how it works, why do you think it is a performance nightmare?